bgpd/bgp_mplsvpn.c

```diff
@@ -964,57 +964,6 @@ void transpose_sid(struct in6_addr *sid, uint32_t label, uint8_t offset,
 	}
 }
 
-static bool labels_same(struct bgp_path_info *bpi, mpls_label_t *label,
-			uint32_t n)
-{
-	uint32_t i;
-
-	if (!bpi->extra) {
-		if (!n)
-			return true;
-		else
-			return false;
-	}
-
-	if (n != bpi->extra->num_labels)
-		return false;
-
-	for (i = 0; i < n; ++i) {
-		if (label[i] != bpi->extra->label[i])
-			return false;
-	}
-	return true;
-}
-
-/*
- * make encoded route labels match specified encoded label set
- */
-static void setlabels(struct bgp_path_info *bpi,
```

Comment:
You remove a function that you have added in a previous commit. Use the final function in the initial commit.